src/cascadia/QueryExtension/ExtensionPalette.cpp

```diff
@@ -489,4 +456,55 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
         // Clear the text box each time we close the dialog. This is consistent with VsCode.
         _queryBox().Text(winrt::hstring{});
     }
+
+    ChatMessage::ChatMessage(winrt::hstring content, bool isQuery, bool isCode) :
+        _messageContent{ content },
+        _isQuery{ isQuery },
+        _isCode{ isCode }
```

Comment:
you'll need a `_richBlock{ nullptr }` here, otherwise we will construct one on line 460 and then overwrite it on line 465.